mdal/frmts/mdal_3di.hpp

```diff
@@ -62,6 +65,13 @@ namespace MDAL
       size_t parse2DMesh();
 
       void addBedElevationDatasetOnFaces();
+
+#ifdef HAVE_SQLITE3
```

Comment:
SQLITE3 is almost always present on all platforms. So instead of `#ifdef HAVE_SQLITE3` all over the place in mdal_3di.hpp and mdal_3di.cpp, just do not compile 3di driver at all when there is no sqlite3 available